improver_tests/utilities/test_GAMPredict.py

```diff
@@ -0,0 +1,50 @@
+# (C) Crown Copyright, Met Office. All rights reserved.
+#
+# This file is part of 'IMPROVER' and is released under the BSD 3-Clause license.
+# See LICENSE in the root of the repository for full licensing details.
+"""Unit tests for the GAMPredict class within statistical.py"""
+
+import numpy as np
+import pytest
+
+from improver.utilities.statistical import GAMPredict
+
+
+@pytest.mark.parametrize(
+    "X_new,expected",
+    [
+        (
+            [[2006, 32, 1]],
+            [92.13292996],
+        ),  # Test prediction of a single new value
```

Comment:
Just for interpretability, I think it might be clearest to add a comment about the dataset used, so that these numbers are more readily understood e.g. 
```suggestion
            [[2006, 32, 1]],  # Year, Age, Education (as a category)
            [92.13292996],  # Wage (in arbitrary units)
        ),  # Test prediction of a single new value
```

Reply:
Added comments to this effect.